src/GitHub.UI/Converters/DurationToStringConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Globalization;
+
+namespace GitHub.UI
+{
+    public class DurationToStringConverter : ValueConverterMarkupExtension<DurationToStringConverter>
+    {
+        public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            const int year = 356;
+            const int month = 30;
+            const int day = 24;
+            const int hour = 60;
+            const int minute = 60;
+
+            TimeSpan duration;
+            if (value is TimeSpan)
+                duration = (TimeSpan)value;
+            else if (value is DateTime)
+                duration = DateTime.Now - (DateTime)value;
```

Comment:
Where would we be getting `value` from. Is it possible it'd be stored as `UTC` and we should use `UtcNow` here? Or can we assume it's always local time?